Week03/lesson06/classwork/startup_matchmaker/Styles/styles.css

```diff
@@ -0,0 +1,182 @@
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+*{
+box-sizing:border-box;
+}
+
+
+nav {
+  list-style-type: none;
+  float:right;
+  display:inline-block;
+  margin: 0.67em 0;
+}
+
+header{
+  background-color: black;
+}
+
+nav a {
+  color: white;
+  text-decoration: none;
+  font-size: 32px;
+}
+
+a:hover{
+  color:yellow;
+}
+
+
+li{
+  display:inline-block;
+  padding: 2.5px;
+}
+
+h1 {
+  color:white;
+  display:inline-block;
+}
+
+nav:last-child{
+  margin-right: 15px;
+}
+
+section{
+  background-color: grey;
+  text-align: center;
+  margin: 0px;
+  border-top: solid yellow 3px;
+  padding: 10px 0px;
+  font-style: italic;
+}
+
+
+main{
+  background-image: url(../images/coworking.jpg);
+  background-size: 100%;
+  height: 600px;
+  background-repeat: no-repeat;
+}
+
+.introbox{
+  background-color: rgba(192,192,192,0.85);
+  float: right;
+  display: inline-block;
+  width: 400px;
+  margin:25px;
+  height:550px;
+}
+
+h3{
+  text-align: center;
+  margin: 0px;
+}
+
+p{
+  margin: 0px 10px;
+}
+
+.style1{
+  text-decoration: line-through;
+}
+
+.style2{
+  text-decoration: underline;
+  text-decoration-color:yellow;
+}
+
+div h3{
+  padding: 25px;
+  font-size: 26px;
+  display: inline-block;
+  text-align: center;
+}
+
+div p{
+  padding: 15px 25px;
+  line-height: 32px;
+  font-size: 22px;
+}
+
+h4 {
+  text-align: center;
+  font-size: 22px;
+}
+
+.profilebutton{
+  background-color: yellow;
+  border-radius: 3px;
+  margin:auto auto;
+  width: 90%;
+  text-align: center;
+  padding: 15px 10px;
+  box-shadow: 0 3px 3px black;
+}
+
+.bottomcontainer{
+  width:80%;
+  margin: auto;
+  font-size: 0;
+}
+
+.column{
+  display:inline-block;
+  width: 30%;
+  margin: 12px;
+}
+
+.column:first-child {
+  border-right: grey 1px solid;
+}
+
+.column:last-child{
+  border-left: grey 1px solid;
+}
+
+footer{
+  background-color: black;
+  color: white;
+  padding: 25px;
+}
+
+.searchbutton{
+  height:35px;
+  background-color: grey;
+  font-size: 18px;
+  box-shadow: 0 3px 3px black;
+  margin: auto;
+  text-align: center;
+  width: 80%;
+  padding: 5px;
+  border-radius: 3px;
+}
+
+.searchcontainer{
+  width:90%;
+  margin:auto;
+  font-size: 0;
+  background-color: #D3D3D3;
+}
+
+.result-box1{
+  width: 25%;
+  display: inline-block;
+  font-size: 12px;
+  background-color: white;
+  padding: 5px;
+  text-align: center;
+}
+
+/*Questions:
+1. How to get rid of extra space on column 3 on index page
+2. How to add margins when using percentages with layout
+3. How to control the box-shadow properties.
+4. How to to change the text decoration properties
```

Comment:
Ahhhh see this is where I was being a bit mean haha. I wanted to see who would think of using `border-bottom` instead when they saw what a pain `text-decoration` is.

In order to do the strikethrough, you definitely want to make use `text-decoration` like you did.

However for the underline, you need to keep a few things in mind: 
1) browsers aren't good at supporting `text-decoration` that's a different colour than the text itself
2) browsers aren't good at allowing separation between the text and the underline
3) browsers aren't good at letting you set a thicker underline than just 1px

So, all in all, it's better to use `border-bottom`. If you apply this CSS:

```
.underline { /* you called this class "style2" in your code, try to go for more descriptive class names!*/
  padding-bottom: 1px;
  border-bottom: 3px solid yellow;
}
```